<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PicklingForm</class>
 <widget class="QWidget" name="PicklingForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="horizontalSpacing">
    <number>30</number>
   </property>
   <property name="verticalSpacing">
    <number>12</number>
   </property>
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <property name="horizontalSpacing">
      <number>9</number>
     </property>
     <property name="verticalSpacing">
      <number>9</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="pa_density_label">
       <property name="text">
        <string notr="true">Density</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="pa_density_lineedit"/>
       </item>
       <item>
        <widget class="QLabel" name="pa_density_unit_label">
         <property name="text">
          <string>kg/m&lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt;</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QFormLayout" name="formLayout_2">
     <property name="horizontalSpacing">
      <number>9</number>
     </property>
     <property name="verticalSpacing">
      <number>9</number>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="pa_displaced_by_label">
       <property name="text">
        <string notr="true">Displaced by</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="pa_displaced_by_lineedit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="pa_additives_label">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <underline>true</underline>
      </font>
     </property>
     <property name="text">
      <string notr="true">Additives</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QTableView" name="pa_additives_tableview">
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
